### DISCUSSION TITLE
Exemplars and metrics (as of 1.9.0)

### DISCUSSION THREAD

**Discussion by pedro-leitao** [open-telemetry/opentelemetry-java#3926] [Q&A]
Hello,

I've gone through the documentation and code, but it is not clear to me what the current state of play is for exemplars and metrics.

Would something like:

counter.add(value, context)

Expose exemplars over the OTLP *and* Prometheus exporter ?

Thanks.

**Comment by jkwatson** [comment]
It definitely works for OTLP. Prometheus exemplars probably work, but there are bugs with it in 1.9.0; we're hoping to get a patch release out very soon to address the issues.

**Reply by jsuereth** [reply]
Prometheus doesn't support exemplars on counters, only histograms.  This should be fixed.  The general principal is we export what we can when it is supported
